sources/packages/backend/apps/db-migrations/src/sql/Student/Add-col-disability-status.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS disability_status sims.disability_status;
+
+COMMENT ON COLUMN sims.students.disability_status IS 'Disability status of the student. The disability could be PD or PPD.';
+
+-- Update the disability_status column based on pd_verified and pd_date_sent for existing data.
+UPDATE
+    sims.students
+SET
+    disability_status = CASE
+        WHEN pd_verified IS NULL
+        AND pd_date_sent IS NOT NULL THEN 'Requested'
+        WHEN pd_verified = true THEN 'PD'
+        WHEN pd_verified = false THEN 'Declined'
+        ELSE 'Not requested' :: sims.disability_status
```

Comment:
I get this update is written for the existing data in Test, but is there a chance that any PDs can be PPDs? are we considering any update in the existing data update from ATBC?

Reply:
There is no PPD in our system so far so we cannot do anything in the query about it.

If a student is PPD, ATBC response will be read and updated in our system.